recipes/quazip/all/test_package/CMakeLists.txt

```
@@ -1,13 +1,10 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package)
+project(test_package CXX)
```

```suggestion
project(test_package LANGUAGES CXX)
```
It is trivial thing. You can miss it.